### ISSUE TITLE
JFoenix tabPane not displayed

### ISSUE THREAD

**Issue by thomaslc66** [jfoenixadmin/JFoenix#597]
Hi everyone,

I'm dealing with a small problem here!

With the last scene Builder, importing the JFoenix library, my JFX Tab Pane are not displayed, i can't even put some other component into them.

Do i need to do someting specific or maybe reimport the library?

Thank you.

**Comment by jfoenixadmin** [comment]
Hi, make sure you are using compatible version of JFoenix / scene builder. 
there are different jars for Java 8 / 9

Regards,

**Comment by thomaslc66** [comment]
Product Version
JavaFX Scene Builder 9.0.1

Build Information
Version 9.0.1
Date: 2017-10-17
Java Version: 9+181, Oracle Corporation

and I imported both jar for java 8 and jar for java 9. Java 9 file don't show any files when importing, java 8 import 41 elements. + I've tested it on my PC and on my Mac, same problem. I even tried to import directly from the "search Jar library" from Scene Builder.

Was expecting a bit more than just this.
